strongbox-data-service/src/main/java/org/carlspring/strongbox/booters/PropertiesBooter.java

```diff
@@ -7,12 +7,14 @@
 /**
  * @author carlspring
  */
+//TODO: SB-1266
+@Deprecated
 public class PropertiesBooter
 {
 
     static
     {
-        initialize();
+        //initialize();
```

Comment:
Okay... but... have you tried this with the `strongbox-distribution`? Is it creating the directories in the right place...? I have the feeling that it's probably creating a mess, by creating the directories wherever it likes... Could you please confirm?

Reply:
I confirm
anyway, all property values was defined in `wrapper.conf` so this `PropertiesBooter` shouldn't affect anything